src/App.tsx

```diff
@@ -1,16 +1,27 @@
 import "./App.css";
 
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+
+import Game from "./pages/Game/Game";
+import Home from "./pages/Home/Home";
+import NoPage from "./pages/NoPage/NoPage";
+
+const getBasename = () => {
+  if (window.location.hostname === "geo-quest.github.io") {
+    return "/travel-bingo";
+  }
+  return "/";
+};
```

Comment:
nitpick: maybe we can solve that outside the code, so the application doesn't need to know anything about the environment, maybe we can use some scripts or environment variables. 

no rush on that, just a note.